src/org-manager/containers/pending-pbas/pending-pbas.component.ts

```diff
@@ -49,6 +52,9 @@ export class PendingPBAsComponent implements OnInit, OnDestroy {
   }
 
   public handleLoadedActiveOrganisations(active: OrganisationVM[], pending: OrganisationModel[]): void {
+    const resetStatus = this.route.snapshot.queryParams.reset;
```

Comment:
I'd probably add an additional note here describing the fact that this is a temporary amend to allow QA and should be removed before merging into feature branch.